### ISSUE TITLE
Position and Size of menu items in 'Create new Workbook' window is not defined

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#393]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to menu item present in menu bar (For e.g. - File->Open) and observe Inspect properties

### Expected Behavior
Position and Size of menu items in 'Create new Workbook' window should be properly defined
### Actual Behavior
Position and Size of menu items in 'Create new Workbook' window is not defined
Narrator Behavior: Narrator is not announcing position and size of menu item and announcing menu item as 'Open, menu item'
NVDA Behavior: NVDA is not announcing position and size of menu item and announcing menu item as 'Open'
Note:This issue is also observed for all menu items
### User Impact
Users who rely on screen reader will not be able to know the position and size of menu item
### Screenshots
![menu_item_position_inspect](https://user-images.githubusercontent.com/22956130/40775535-f40df542-64e5-11e8-9e4c-1daa930e6ad7.png)
![menu_item_position_narrator](https://user-images.githubusercontent.com/22956130/40775536-f43d2e02-64e5-11e8-84db-20e31c5f0051.png)
![menu_item_position_nvda](https://user-images.githubusercontent.com/22956130/40775537-f4735dc4-64e5-11e8-8f1e-53ee910a6c39.png)

      
### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yUsable
Accessibility
Xamarin-Workbooks-Windows
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
